src/applications/disability-benefits/686c-674-v2/config/chapters/report-add-child/config.js

```diff
@@ -0,0 +1,18 @@
+export const arrayBuilderOptions = {
+  arrayPath: 'childrenToAdd',
+  nounSingular: 'child',
+  nounPlural: 'children',
+  required: true,
+  isItemComplete: _item => {
+    return true;
```

Comment:
`isItemComplete` has to be hooked up to each individual `required` and conditionally `required` field instead of being defaulted to true. See config options in `686c-674-v2/config/chapters/674/addStudentsArrayPages.js`